### ISSUE TITLE
About the evaluation code.

### ISSUE THREAD

**Issue by Yangr116** [Q-Future/Q-Instruct#12]
Hi, thanks for your insightful work, I would like to know why are you use the last logits to calculate score?
![image](https://github.com/Q-Future/Q-Instruct/assets/73805072/c11f7a47-248d-463a-95c3-9311228e4556)

**Comment by teowu** [comment]
Hi Rui,

This is the logic as proposed by Q-Bench.

![](https://raw.githubusercontent.com/Q-Future/Q-Bench/master/llmiqa.png)

 For Q-Instruct, as IQA is not the pre-trained goal for it, we follow this evaluation strategy.

**Comment by Yangr116** [comment]
Thanks for your quick reply. According to this design, does that mean the score token equals the eos token?

**Comment by teowu** [comment]
Not actually. This is actually the first token MLLM responses after the pre-set starting words.

**Comment by Yangr116** [comment]
Got it. Thanks!
